yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/SecureYotiClient.java

```diff
@@ -141,13 +150,13 @@ private void validateReceipt(Receipt receipt) throws ActivityFailureException {
     }
 
     private Profile createProfile(byte[] profileBytes, Key secretKey) throws ProfileException {
-        Map<String, Object> attributeMap = new HashMap<String, Object>();
+        List<com.yoti.api.client.Attribute> attributeList = new ArrayList<com.yoti.api.client.Attribute>();
```

Comment:
I know we have 2x __Attribute__ classes in play, but if we use __Attribute__ and __AtributeProto.Attribute__ then we can avoid these fully qualified class names